webcamd.c

```diff
@@ -691,8 +691,14 @@ pidfile_create(int bus, int addr, int index)
 	if (local_pid != NULL)
 		return (0);
 
-	snprintf(buf, sizeof(buf), "/var/run/webcamd."
-	    "%d.%d.%d.pid", bus, addr, index);
+	if (do_v4l2loopback == 0) {
+		snprintf(buf, sizeof(buf), "/var/run/webcamd."
+		    "%d.%d.%d.pid", bus, addr, index);
+	} else {
+		snprintf(buf, sizeof(buf), "/var/run/webcamd."
+		    "v4l2loopback.pid", bus, addr, index);
```

Comment:
Please remove those additional arguments, bus, addr, index for the loopback case!

Reply:
Whoops! Sorry, fixed.